culturefeed_agenda/theme/theme.inc

```diff
@@ -1643,14 +1643,18 @@ function theme_culturefeed_agenda_page_list_rss($variables) {
       );
 
       $body = '';
+      $body .= "<p>";
```

Comment:
Zijn we hier zeker dat er nooit geen <p> tags in de "schort_description" zullen zitten? Zoniet zal er foutieve html uitgespuwd worden.